src/libostree/ostree-gpg-verifier.c

```diff
@@ -304,12 +304,85 @@ _ostree_gpg_verifier_add_key_ascii_file (OstreeGpgVerifier *self,
   g_ptr_array_add (self->key_ascii_files, g_strdup (path));
 }
 
+gboolean
+_ostree_gpg_verifier_add_keyfile_path (OstreeGpgVerifier   *self,
+                                       const char          *path,
+                                       GCancellable        *cancellable,
+                                       GError             **error)
+{
+  GError *temp_error = NULL;
```

Comment:
The pattern we normally use for this is `g_autoptr(GError)` and then `g_propagate_error (error, g_steal_pointer (&temp_error))`.

Reply:
Addressed in https://github.com/ostreedev/ostree/pull/1773/commits/1e120d2a41107b9e128e1bed2820eceb080a69fb